build_docs.pl

```diff
@@ -125,7 +127,7 @@ sub build_local {
 
     if ( $Opts->{open} ) {
         say "Opening: " . $html;
-        serve_and_open_browser( $dir );
+        serve_and_open_browser( $dir, 0 );
     }
```

Comment:
The second argument is `$redirects_file`. Isn't that a string? What strange autocast magic is happening here?

Reply:
Think of `0` as Python's `None` or Ruby's `nil`. `$redirects_file` is "anything that's string representation looks like a file or anything falsy 0 to skip the file." Let me document the parameters.

Reply:
Ah cool. It's been a looong time since I wrote much Perl. If that's a common falsy/nulley value in Perl then I'm good.